conversant/demo/streamlit_example.py

```diff
@@ -128,6 +127,26 @@ def update_prompt_from_json() -> None:
                 0, f"from launch_demo: {st.session_state.bot.persona_name}"
             )
 
+    # Adding a header to direct users to sign up for Cohere, explore the playground,
+    # and check out our git repo.
+    st.header("🎭 Conversational personas using Cohere")
+    with st.expander("About", expanded=True):
+        st.markdown(
+            """
+        This demo app is using 
+        [conversant](https://github.com/cohere-ai/sandbox-conversant-lib), an 
+        open-source framework for building chatbots on top of Cohere’s large 
+        language models. 
+
+        Cohere provides access to advanced Large Language Models and NLP tools through 
+        one easy-to-use API. 
+        [Get started for free!](https://dashboard.cohere.ai/welcome/register)
```

Comment:
@neilAtCohere do you have a recommendation on what link to use, perhaps with a utm param/

Reply:
I'll tag him on Slack to get his sign off on the copy, I believe he may be less active on github.